src/extractor/extraction_containers.cpp

```diff
@@ -453,11 +453,13 @@ void ExtractionContainers::PrepareEdges(lua_State *segment_state)
                 all_edges_list[i].result.weight < min_forward_weight)
             {
                 min_forward_idx = i;
+                min_forward_weight = all_edges_list[i].result.weight;
```

Comment:
this value is never read? And the IDX is used to choose only the minimal edge. All others are invalidated. Does this even affect anything?